src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1435,3 +1435,45 @@ function CategoricalArrays.categorical(df::AbstractDataFrame,
     end
     DataFrame(newcols, _names(df), copycols=false)
 end
+
+"""
+    flatten(df::AbstractDataFrame, veccol::Union{Integer, Symbol})
```

Comment:
The general convention is to call the second argument `cols`, and allow it to point either to a single or to multiple columns. Better use the same naming convention, even if you don't want to support multiple columns for now (would that be hard?).

Reply:
I considered making support for multiple `cols` but in this case it did not make sense for me (would we do a cross-product of these cols)?

Reply:
Still - I agree that having the names consistent is better.